Projects/UOContent/Multis/Boats/ConfirmDryDockGump.cs

```diff
@@ -1,40 +1,43 @@
 using Server.Gumps;
 using Server.Network;
 
-namespace Server.Multis
+namespace Server.Multis;
+
+public class ConfirmDryDockGump : StaticGump<ConfirmDryDockGump>
 {
-    public class ConfirmDryDockGump : Gump
+    private readonly BaseBoat _boat;
+    private readonly Mobile _from;
```

Comment:
For these gumps, we don't even need to store the `_from`. Let's use `state.Mobile` (when it is not null).